### ISSUE TITLE
it throw error with the new version react-spring, react-spring is in beta and interface changed!

### ISSUE THREAD

**Issue by heerzhang** [bmcmahen/toasted-notes#16]

**Comment by bmcmahen** [comment]
So weird... it's working for me when I clone this repo and use storybook. Can you try the same? Out of curiosity, and you using yarn or npm?

**Comment by heerzhang** [comment]
I use Sancho + react-spring, while Sancho use toasted-notes,  it was nested dependency.  toasted-notes can move to the new version react-spring ?  My projecct use "react-spring": "^9.0.0-beta.31",

**Comment by TadewosBell** [comment]
has there been a fix?

**Comment by TadewosBell** [comment]
i am getting
`Cannot read property 'ref' of null`

on 
Message.js line 68
`const transition = useTransition(localShow, null, animation);`

**Comment by LiteSoul** [comment]
Yes, it trows error re. react-spring if you try to add toasted-notes.
I'm using Yarn.
And this is a new project.

![toast-error](https://user-images.githubusercontent.com/22918889/68715413-b6db4e80-0580-11ea-9573-72a649fa9fbb.png)

**Comment by bmcmahen** [comment]
`react-spring` is now listed as a peer dependency, so currently you need to install it yourself. 

Unfortunately I just don't have much time to work on this repo recently. I'd eventually like to just rewrite it - made the mistake of switching to the react-spring 9 beta, which has resulted in some of these dependency problems.

**Comment by heerzhang** [comment]
I change  react-spring to 8.0.27 in both sancho and my Project, but sill get error：
 Scroll to{ }?  or, web.js of react-spring;

**Comment by heerzhang** [comment]
my package.json: :   
 "name": "frontend-sei",
    "dependencies": {
        "react-spring": "8.0.27",
        "sancho   || customize-easy-ui-component" : "3.5.7-hez04",
        "react-gesture-view": "^2.1.3",
        "react-gesture-stack": "^1.3.2",
    },
    "peerDependencies": {
    },

**Comment by LiteSoul** [comment]
> `react-spring` is now listed as a peer dependency, so currently you need to install it yourself.
> 
> Unfortunately I just don't have much time to work on this repo recently. I'd eventually like to just rewrite it - made the mistake of switching to the react-spring 9 beta, which has resulted in some of these dependency problems.

Yes, installing react-spring solves the problem. Sad to hear you don't have time lately

**Comment by TadewosBell** [comment]
how did you solve this issue? i have production web app that depends on this and keeps breaking.

**Comment by bmcmahen** [comment]
have you installed react-spring v8 as a peer dependency?

**Comment by TadewosBell** [comment]
yes, that solved the issue.Is there a way to stop it from breaking if i run npm update

**Comment by zachintosh** [comment]
I am on version 9 (latest beta) and am currently having this issue. Has there been any other issues related to this that might point me to a solution?
